### DISCUSSION TITLE
Is there a way to make boxart images larger or a fixed size.

### DISCUSSION THREAD

**Discussion by artemu5** [flyinghead/flycast#995] [Q&A]
I usually have flycast window maximized and boxart covers are very small, it's hard to read the titles or see the artwork. I'd like to make them at least 3-4x larger. Is there any way to do this?

![flycast-window-crop](https://user-images.githubusercontent.com/126365143/228096841-c18f254a-2a2b-4f34-84a0-99207103c2ce.png)

**Comment by munashige** [comment]
Nope. But I'd like to see this feature implemented too.

**Comment by flyinghead** [comment]
This is available on the dev branch: Settings > General > UI Scaling
